src/componants/CommentsCard.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState, useEffect } from 'react';
+
+const CommentsCard = (item) => 
+{
+    console.log(item.item)
+    return <>
+    <h3>{item.item.author}</h3>
+    <p>Posted at {item.item.created_at}. it has {item.item.votes} votes. </p>
```

Comment:
The created at just gives you a long string, you could see if you could make it more readable for the user. There are libraries you could use that do this, worth a google! Or do something manually